src/main/java/core/basesyntax/model/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.model;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public String getRandomColor() {
+        String[] randomColor = {"Blue", "White", "Green", "Red", "Yellow", "Black"};
```

Comment:
lets make colors as constant
```suggestion
        String[] colors = {"Blue", "White", "Green", "Red", "Yellow", "Black"};
```

Reply:
давай переименуем randomColor на colors и сделаем это поле константой. Как это делается можешь посмотреть тут: https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names